utils/oscap-xccdf.c

```diff
@@ -139,6 +139,7 @@ static struct oscap_module XCCDF_EVAL = {
 		"INPUT_FILE - XCCDF file or a source data stream file\n\n"
         "Options:\n"
         "   --profile <name>\r\t\t\t\t - The name of Profile to be evaluated.\n"
+	"   --rule <name>\r\t\t\t\t - The name of a single rule to be evaluated.\n"
```

Comment:
@jan-cerny please note that lines surrounding this addition are indented by spaces, thus using tab in this particular context is wrong, imho. Either fix it in whole block, or use what is used, and fix it later in separate commit. Sorry for not justifying my objection.